web/satellite/src/components/notifications/NotificationArea.vue

```diff
@@ -17,27 +17,25 @@
         }
     })
     export default class NotificationArea extends Vue {
-        public get currentNotification(): Notification {
-            return this.$store.getters.currentNotification;
+        public get notifications(): Notification {
```

Comment:
Not clear to me. We return Notification Component?
According to store, we return []DelayedNotification, return type for this method is Notification